source/frontend/src/features/properties/map/activity/detail/ActivityPropertyModal.tsx

```diff
@@ -58,16 +61,19 @@ export const ActivityPropertyModal: React.FunctionComponent<
   };
 
   const handleOk = async () => {
-    if (activityModel && allProperties?.length > 0) {
-      activityModel.actInstPropFiles =
-        selectedFileProperties.map(p => ({
-          activityId: activityModel?.id,
-          propertyFileId: p?.id,
-        })) ?? [];
-      const updatedActivity = await onSave(activityModel);
-      // if the activity was updated successfully, hide the modal.
-      if (!!updatedActivity) {
-        setDisplay(false);
+    if (disableSave) setDisplay(false);
```

Comment:
I think we wrap `if` statements with braces, even for one-liners; ie

```tsx
if (...) {
 setDisplay(false)
}
```

Reply:
correct